### PR TITLE
Update the list  of industry options

### PR DESCRIPTION
**Pull request by esloho**
Related to https://github.com/CartoDB/cartodb-central/issues/2589

To be merged right after Hubspot's list has been updated.

### CODE REVIEW

File: lib/assets/javascripts/dashboard/views/profile/profile-form/profile-form-view.js [hunk 1]
```diff
@@ -23,10 +23,11 @@ const REQUIRED_OPTS = [
...
 const INDUSTRIES = [
-  'Apparel & Fashion', 'Banking & Financial Services', 'Business Services', 'Consulting Services',
-  'Consumer & Retail', 'Education & Research', 'Energy & Mining', 'Government', 'Health & Medical', 'Insurance',
-  'Manufacturing', 'Marketing & Advertising', 'Natural Resources & Environment', 'Non-Profit', 'Other', 'Real Estate',
-  'Software & Technology', 'Transportation & Logistics', 'Utilities & Communications'
+  'Architecture, Engineering and Construction', 'Banking and Financial Services', 'Cities and Government',
+  'Consulting', 'CPG', 'Education and Research', 'Health and Medical', 'Insurance', 'Journalism and Media',
+  'Manufacturing', 'Marketing and Advertising', 'Mining', 'Natural Resources and Environment', 'Non-Profit',
+  'Real Estate', 'Retail', 'Software and Tech', 'Sports and Entertainment', 'Telecommunications',
+  'Transport and Logistics', 'Utilities', 'Other'
```
Comment (gonzaloriestra) [inline lib/assets/javascripts/dashboard/views/profile/profile-form/profile-form-view.js]: I think we should sort the list alphabetically

Comment (esloho) [reply lib/assets/javascripts/dashboard/views/profile/profile-form/profile-form-view.js]: I thought it was! But I see the "Other" is the one misplaced. Should we move it then?

**Comment by esloho** [comment]
> It seems like there's a test counting the number of options in industries, and it changed.

Yep, but the weird thing is the number is getting now... it doesn't make sense to me xDDD It was expecting 21 and now it gets 24... but I only count 22!!! I'm still investigating :microscope:

**Comment by esloho** [comment]
OK, so I see the test counts 2 extra elements in the dropdown. Before it counted 21 for 19 industry options so now that we have 22 it does make sense to count 24 :D. Mistery solved! (partially, I still don't know where the 2 extra elements come from. One for "Select one" caption and the other...?)

File: lib/assets/javascripts/dashboard/views/profile/profile-form/profile-form-view.js (hunk 1 above)
Comment (gonzaloriestra) [reply lib/assets/javascripts/dashboard/views/profile/profile-form/profile-form-view.js]: Mmm I think I prefer to have it ordered, but it maybe makes sense to have `Other` separated... Not sure!

@newLadyAga what do you think? This is the list we show in the UI to select the industry. Should it be ordered or should we place `Other` at the bottom?

Comment (newLadyAga) [reply lib/assets/javascripts/dashboard/views/profile/profile-form/profile-form-view.js]: Hi!
Could we have it sorted alphabetically but with `Other` at the end?
Thanks!

Comment (gonzaloriestra) [reply lib/assets/javascripts/dashboard/views/profile/profile-form/profile-form-view.js]: Yes, this is how we have it here. Thank you! :slightly_smiling_face:

Comment (newLadyAga) [reply lib/assets/javascripts/dashboard/views/profile/profile-form/profile-form-view.js]: Perfect. 
Thank you! 🙂

**Pull request merged**